browser/js/bundle.entry.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  bel: require("bel"),
+  morphdom: require('morphdom'),
```

Comment:
Sigh, indeed. Any feelings on which quotes are more progressive in JS these days?